### ISSUE TITLE
Feature Request: Make essence an ingredient for Cauldrons

### ISSUE THREAD

**Issue by Inframission** [CivClassic/AnsibleSetup#81]
Essence currently serves no purpose but to fuel pearls, and while functional the change from mana to essence has only made it easier to keep players imprisoned by enabling owners to store it. While a market for essence exists, giving it multiple uses (e.g. making it an ingredient) would make essence more of a commodity, one that players could make decisions about how to use.

In short by raising demand we can both indirectly nerf imprisonment and also raise the value of essence for players.

**Comment by Maxopoly** [comment]
So active ingredient for big XP runs or only for cauldron creation?

**Comment by Inframission** [comment]
Hope to include this as part of the input for runs, not just for factory creation.

**Comment by KaloHG** [comment]
I feel more as if it being used to repair factories such as the ore smelter and EXP factories, as well as initial factory creation would be better than directly creating recipes that require essence. It would really hinder new players when they try to get things done if they need to pay essence per factory run.

**Comment by wingzero54** [comment]
I don't think this will ever be a feature. I appreciate the thought, but I would rather make adjustments to essence acquisition / pearl cost rather than make pearl fuel be used for other things. I think this comes from a similar mind space as CivRealms having Stamina with multiple uses, but I don't think that fits the vibe and history of CivClassic

**Comment by Inframission** [comment]
Players can always choose to store essence forever but as it stands

**Comment by TheRealArchie** [comment]
I probably shouldn't be apart of this discussion but I as a player wouldn't mind either way. I do like the arguments infra makes :)
